BlueprintUI/Sources/Element/ElementState.swift

```diff
@@ -235,12 +251,14 @@ final class ElementState {
         identifier: ElementIdentifier,
         element: Element,
         signpostRef: AnyObject,
-        name: String
+        name: String,
+        kind: Kind = .regular
```

Comment:
IMO given this is an internal-only method, I think requiring folks to pass a value is best, vs them forgetting and getting it wrong